src/utils.ts

```diff
@@ -1,32 +1,105 @@
   return moment(date).format('YYYYMMDD_HHmm')
 }
 
-export function parseYAML(filePath: string, repo: string | undefined, content: string): any {
+export function parseYAML(
+  filePath: string,
+  repo: string | undefined,
+  content: string
+): any {
   const defaultValue = 'Undefined' // Default value when json field is not defined
   let name = defaultValue
   let author = defaultValue
   let description = defaultValue
 
   try {
-    const parsed = YAML.parse(content)    
+    const parsed = YAML.parse(content)
     name = parsed.name ? sanitize(parsed.name) : defaultValue
     author = parsed.author ? sanitize(parsed.author) : defaultValue
-    description = parsed.description ? sanitize(parsed.description) : defaultValue
+    description = parsed.description
+      ? sanitize(parsed.description)
+      : defaultValue
   } catch (error) {
     // this happens in https://github.com/gaurav-nelson/github-action-markdown-link-check/blob/9de9db77de3b29b650d2e2e99f0ee290f435214b/action.yml#L9
     // because of invalid yaml
-    core.warning(`Error parsing action file [${filePath}] in repo [${repo}] with error: ${(error as Error)}`)
-    core.info(`The parsing error is informational, seaching for actions has continued`)    
+    core.warning(
+      `Error parsing action file [${filePath}] in repo [${repo}] with error: ${
+        error as Error
+      }`
+    )
+    core.info(
+      `The parsing error is informational, seaching for actions has continued`
+    )
   }
-  return { name, author, description }
+  return {name, author, description}
 }
 
 function sanitize(value: string) {
   return string.sanitize.keepSpace(value)
 }
+
+// Interface for a Dockerfile with actionable properties
+export interface DockerActionFiles {
+  [key: string]: string | undefined
+  name?: string
+  description?: string
+  // icon?: string
+  // color?: string
+
+  // Icon and color is needed for using dockerfiles as actions, but it's not used in the marketplace.
+}
+
+export const returnActionableDockerFiles = async (path: string) => {
+  const dockerFilesWithActionArray: DockerActionFiles[] = []
+  const dockerFiles = execSync(
+    `find ${path} -name "Dockerfile" -o -name "dockerfile"`,
+    {encoding: 'utf8'}
+  ).split('\n')
+  // Asynchronously process each Dockerfile
+  await Promise.all(
+    dockerFiles.map(async (item: string) => {
+      // If the Dockerfile path is valid
+      if (item) {
+        // Remove the "actions/${path}/" prefix from the path
+        item = item.replace(`actions/${path}/`, '')
+
+        try {
+          // Read the contents of the Dockerfile
+          const data = await promisify(fs.readFile)(item, 'utf8')
+          const label = 'LABEL com.github.actions.'
+          // Check if the Dockerfile has actionable properties
+          if (
+            data.includes(`${label}name=`) &&
+            data.includes(`${label}description=`)
+          ) {
+            core.info(`${item} has dockerfile as an action!`)
```

Comment:
Add brackets to match the rest of the code
```suggestion
            core.info(`[${item}] has dockerfile as an action!`)
```

Reply:
Done